### ISSUE TITLE
Tests for twisted.trial should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7877]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7877|
|Type|enhancement|
|Created|2015-04-29 14:19:18Z|
|Branch|https://github.com/twisted/twisted/tree/test-names-trial-7877|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [misc_7877.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/009/0090d1eadb0ffd834f02eeeb1634f1919e016dfc/e7d6eaefc64ccaa31ab4931d0abe1139580b3db0.patch) (42049 bytes) - added by eeshangarg on 2015-04-29 14:21:55Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7877 7877
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_test_names_trial_7877 branches/test-names-trial-7877
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__easy easy
time__1430317158823468 1430317158823468
changetime__1430324467262965 1430324467262965
version__None None
owner__ 
cc__eeshangarg cc__jml
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_2"></a>|@eeshangarg removed owner|
|-|-|

Hi!

I just added a patch for this called misc_7877.patch. I think this is the last component of Twisted that needed it's test class names to conform to the test class naming convention recommended in our coding standard. However, if you do find some other test classes that are still left in any other component, please let me know! I would be willing to work on it. :-)

Thanks,
Eeshan Garg

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

(In [44620]) Branching to test-names-trial-7877.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set status to `closed`|
|-|-|

(In [44622]) Merge test-names-trial-7877: Update test in twisted.trial to conform to the new style of naming test classes.

Author: eeshangarg
Reviewers: adiroiban
Fixes: [#7877](https://github.com/twisted/twisted/issues/7877)

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban commented|
|-|-|

Thanks!

Changes merge... next step would be to update twisted checker https://github.com/twisted/twistedchecker to raise an error when we try to merge new code which does not comply with this convention.
